SonarScanner.Shim/PropertiesFileGenerator.cs

```diff
@@ -213,7 +279,8 @@ private static string GetSourceEncoding(AnalysisProperties properties, IEncoding
         {
             try
             {
-                if (Property.TryGetProperty(SonarProperties.SourceEncoding, properties, out Property encodingProperty))
+                Property encodingProperty;
+                if (Property.TryGetProperty(SonarProperties.SourceEncoding, properties, out encodingProperty))
```

Comment:
Why did you revert C# 7 change?

Reply:
Probably I missed that... I copy-pasted my files on top of JH's changes and then manually reverted changes.